app/router.tsx

```diff
@@ -1,22 +1,53 @@
 import { createRouter as TanStackCreateRouter } from '@tanstack/react-router'
+import { routerWithQueryClient } from '@tanstack/react-router-with-query'
+import { ConvexQueryClient } from '@convex-dev/react-query'
+import { ConvexProvider } from 'convex/react'
 import { routeTree } from './routeTree.gen'
 import { DefaultCatchBoundary } from './components/DefaultCatchBoundary'
 import { NotFound } from './components/NotFound'
+import { QueryClient } from '@tanstack/react-query'
 
 export function createRouter() {
-  const router = TanStackCreateRouter({
-    routeTree,
-    defaultPreload: 'intent',
-    defaultErrorComponent: DefaultCatchBoundary,
-    defaultStaleTime: 1,
-    defaultNotFoundComponent: () => {
-      return <NotFound />
-    },
-    context: {
-      assets: [],
+  const CONVEX_URL =
+    (import.meta as any).env.VITE_CONVEX_URL ||
+    'https://intent-pigeon-358.convex.cloud'
```

Comment:
maybe a comment about what this instance is and who owns it?

Reply:
Did this, not sure why GitHub isn't showing the change here.